handlers/officerDeltaHandler.go

```diff
@@ -1,22 +1,45 @@
 package handlers
 
 import (
+	"github.com/companieshouse/chs-delta-api/helpers"
+	"github.com/companieshouse/chs-delta-api/services"
 	"net/http"
 )
 
-// OfficerDeltaHandler offers a handler by which to publish a message onto a kafka topic.
+const (
+	officersTopic = "officers-delta"
```

Comment:
We normally follow this naming convention for constants in GoLang where we start with a capital letter and then follow camel case
```suggestion
	OfficersTopic = "officers-delta"
```

Reply:
I will look for all constants and change to fit this naming convention